bench/wtperf/wtperf.c

```diff
@@ -1540,8 +1540,7 @@ execute_populate(CONFIG *cfg)
 		print_ops_sec = 0;
 	} else {
 		print_secs = (double)msecs / (double)MSEC_PER_SEC;
-		print_ops_sec =
-		    (uint64_t)((cfg->icount / msecs) / MSEC_PER_SEC);
+		print_ops_sec = (uint64_t)(cfg->icount / print_secs);
```

Comment:
Note that this is unrelated but has been bugging me -- at the end of a populate, `wtperf` will now print a meaningful "ops/sec" result.